src/app/components/core/configuration-manager/configuration-group/configuration-group.component.html

```diff
@@ -1,56 +1,70 @@
+<div class="container">
+  <div class="configuration_group_wrapper">
+    <nav id="group_navigation" class="groupNav dragscroll mouse-scroll" role="tablist">
+      <div id="groupNavContents" class="nav groupNav_contents">
+        <div class="tabs is-boxed" *ngIf="from !== 'config-manager'">
+          <ul>
+            <li *ngFor="let g of groups" [ngClass]="{'is-active': selectedGroup == g.group}">
+              <h6 class="is-6 has-text-weight-semibold">
+                <a (click)="selectTab(g.group)">
+                  {{g.group}}
+                </a>
+              </h6>
+            </li>
+            <!-- Advanced and security group -->
+            <li *ngFor="let g of advanceCategoriesGroup" [ngClass]="{'is-active': selectedGroup == g.group}">
+              <h6 class="is-6 has-text-weight-semibold">
+                <a (click)="selectTab(g.group)">
+                  {{g.group}}
+                </a>
+              </h6>
+            </li>
+            <ng-container *ngIf="from !== 'notification'">
+              <li [ngClass]="{'is-active': selectedGroup == 'developer'}"
+                *ngIf="developerFeaturesService.getDeveloperFeatureControl() && pages.includes(from) && rolesService.hasEditPermissions()">
+                <h6 class="is-6 has-text-weight-semibold">
+                  <a (click)="selectedGroup = 'developer'">Developer</a>
+                </h6>
+              </li>
+            </ng-container>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <button (click)="left()" id="leftScroll" class="advancer advancer_left" type="button">
+      <i class="fas fa-angle-left fa-xl"></i>
+    </button>
+    <button (click)="right()" id="rightScroll" class="advancer advancer_right" type="button">
+      <i class="fas fa-angle-right fa-xl"></i>
+    </button>
+  </div>
+  <div [ngClass]="{'highlight': from !== 'config-manager'}">
+    <ng-container *ngFor="let g of groups">
       <app-show-configuration [hidden]="selectedGroup !== g.group" [groupConfiguration]="g.config" [group]="g.group"
-        [selectedGroup]="selectedGroup" [fullConfiguration]="g.config" (event)="getChangedAdvanceConfiguration($event)"
+        [selectedGroup]="selectedGroup" [fullConfiguration]="category.config" (event)="getChangedConfiguration($event)"
         (formStatusEvent)="formStatus($event)"></app-show-configuration>
     </ng-container>
-    <ng-template #SecurityGroup>
-      <ng-container *ngIf="g.category == category?.name+'Security'; else pluginGroup">
+    <!-- Advanced and security group configuration -->
+    <ng-container *ngFor="let g of advanceCategoriesGroup">
+      <ng-container *ngIf="g.category == category?.name+'Advanced'; else SecurityGroup">
         <app-show-configuration [hidden]="selectedGroup !== g.group" [groupConfiguration]="g.config" [group]="g.group"
           [selectedGroup]="selectedGroup" [fullConfiguration]="g.config"
-          (event)="getChangedSecurityConfiguration($event)"
```

Comment:
getChangedSecurityConfiguration is not required? this fix is for develop too?

Reply:
this was a mistake, fixed this in latest commit